packages/cache/src/internal/cacheUtils.ts

```diff
@@ -82,19 +83,33 @@ async function getVersion(app: string): Promise<string> {
 
 // Use zstandard if possible to maximize cache performance
 export async function getCompressionMethod(): Promise<CompressionMethod> {
+  if (process.platform === 'win32' && !isGnuTarInstalled()) {
```

Comment:
I'm fine with not messing with zstd on Windows if we want to be consistent across Windows versions.